code/modules/antagonists/changeling/changeling.dm

```diff
@@ -916,24 +954,43 @@
 	/// ID HUD icon associated with the profile
 	var/id_icon
 	
-	/// SKYRAT EDIT START
+	/// SKYRAT EDIT START	( TODO: Move some of this upstream. )
+	/// The colour of the underwear worn by the profile source.
 	var/underwear_color
+	/// The colour of the undershirt worn by the profile source.
 	var/undershirt_color
+	/// The colour of the socks worn by the profile source.
 	var/socks_color
+	/// The colour of the left eye of the profile source.
 	var/eye_color_left
+	/// The colour of the right eye of the profile source.
 	var/eye_color_right
+	/// Whether the profile source had emissive eyes.
 	var/emissive_eyes
+	/// The hair and facial hair gradient styles of the profile source.
 	var/list/grad_style = list("None", "None")
+	/// The hair and facial hair gradient colours of the profile source.
 	var/list/grad_color = list(null, null)
+	/// The body type (male/female) of the profile source.
 	var/physique
+	/// Digitigrade icons of the items/clothes worn by the profile source.
 	var/list/worn_icon_digi_list = list()
+	/// Teshari icons of the items/clothes worn by the profile source.
 	var/list/worn_icon_teshari_list = list()
+	/// Vox icons of the items/clothes worn by the profile source.
 	var/list/worn_icon_vox_list = list()
+	/// Variation flags of the items/clothes worn by the profile source.
 	var/list/supports_variations_flags_list = list()
+	/// The typepath of the selected_scream used by the profile source.
 	var/scream_type
+	/// The typepath of the selected_laugh used by the profile source.
 	var/laugh_type
+	/// The age of the profile source.
 	var/age
+	/// The quirks of the profile source.
 	var/list/quirks = list()
+	/// Whether the profile source displayed a gun permit icon when looked at with a sechud.
+	var/gun_permit
 	/// SKYRAT EDIT END
```

Comment:
This should likely also all be moved to a modular file instead of being added here non-modularly.